drivers/adcxx1c/adcxx1c.c

```diff
@@ -116,23 +130,42 @@ int adcxx1c_set_alert_parameters(const adcxx1c_t *dev, int16_t low_limit,
                                  int16_t high_limit, int16_t hysteresis)
 {
     uint8_t buf[2];
+    int status;
 
     i2c_acquire(I2C);
 
     low_limit <<= (12 - dev->params.bits);
     buf[0] = low_limit >> 8;
     buf[1] = low_limit & 0xFF;
-    i2c_write_regs(I2C, ADDR, ADCXX1C_LOW_LIMIT_ADDR, buf, 2, 0);
+    status = i2c_write_regs(I2C, ADDR, ADCXX1C_LOW_LIMIT_ADDR, buf, 2, 0);
+    if (status < 0) {
+        i2c_release(I2C);
+        LOG_DEBUG("[adcxx1c] set_alert (low limit) - error: unable to communicate "
+                  "with the device (err=%d)\n", status);
+        return ADCXX1C_NOI2C;
+    }
 
     high_limit <<= (12 - dev->params.bits);
     buf[0] = high_limit >> 8;
     buf[1] = high_limit & 0xFF;
-    i2c_write_regs(I2C, ADDR, ADCXX1C_HIGH_LIMIT_ADDR, buf, 2, 0);
+    status = i2c_write_regs(I2C, ADDR, ADCXX1C_HIGH_LIMIT_ADDR, buf, 2, 0);
+    if (status < 0) {
+        i2c_release(I2C);
+        LOG_DEBUG("[adcxx1c] set_alert (high limit) - error: unable to communicate "
+                  "with the device (err=%d)\n", status);
+        return ADCXX1C_NOI2C;
+    }
 
     hysteresis <<= (12 - dev->params.bits);
     buf[0] = hysteresis >> 8;
     buf[1] = hysteresis & 0xFF;
-    i2c_write_regs(I2C, ADDR, ADCXX1C_HYSTERESIS_ADDR, buf, 2, 0);
+    status = i2c_write_regs(I2C, ADDR, ADCXX1C_HYSTERESIS_ADDR, buf, 2, 0);
+    if (status < 0) {
+        i2c_release(I2C);
+        LOG_DEBUG("[adcxx1c] set_alert (hysteresis) - error: unable to communicate "
```

Comment:
What's the benefit of using LOG_DEBUG instead of DEBUG ? Is there a consensus or some guidelines if/when one should be used instead of the other ?

Reply:
It was more convenient for me because we use `LOG_*` everywhere and it's enabled for the whole app (not at a file level like `DEBUG`). I'm not aware of any consensus or guidelines for that, except for saul auto-init code which seems to use ` LOG_*`. I can revert this to `DEBUG` if you prefer.

Reply:
I would prefer using the DEBUG, just for consistency. I haven't checked in details but I think this is what is used in general in driver implementations.

Reply:
Fixed